vaadin-spring/src/main/java/com/vaadin/flow/spring/VaadinConfigurationProperties.java

```diff
@@ -112,39 +112,84 @@ public void setPnpmEnabled(boolean pnpmEnabled) {
     }
 
     /**
-     * Get a list of packages that are blacklisted for class scanning.
+     * Get a list of packages that are blocked for class scanning.
      *
-     * @return package blacklist
+     * @return blocked packages
+     * @deprecated use getBlockedPackages()
      */
+    @Deprecated(forRemoval = true)
     public List<String> getBlacklistedPackages() {
-        return Collections.unmodifiableList(blacklistedPackages);
+        return Collections.unmodifiableList(blockedPackages);
     }
 
     /**
-     * Set list of packages to ignore for class scanning.
+     * Get a list of packages that are blocked for class scanning.
      *
-     * @param blacklistedPackages list of packages to ignore
+     * @return blocked packages
      */
-    public void setBlacklistedPackages(List<String> blacklistedPackages) {
-        this.blacklistedPackages = new ArrayList<>(blacklistedPackages);
+    public List<String> getBlockedPackages() {
+        return Collections.unmodifiableList(blockedPackages);
     }
 
     /**
-     * Get a list of packages that are white-listed for class scanning.
+     * Set a list of packages to ignore for class scanning.
      *
-     * @return package white-list
+     * @param blockedPackages list of packages to ignore
+     * @deprecated use setBlockedPackages()
      */
+    @Deprecated(forRemoval = true)
+    public void setBlacklistedPackages(List<String> blockedPackages) {
+        this.blockedPackages = new ArrayList<>(blockedPackages);
+    }
+
+    /**
+     * Set a list of packages to ignore for class scanning.
+     *
+     * @param blockedPackages list of packages to ignore
+     */
+    public void setBlockedPackages(List<String> blockedPackages) {
+        this.blockedPackages = new ArrayList<>(blockedPackages);
+    }
+
+    /**
+     * Get a list of packages that are allowed for class scanning.
+     *
+     * @return allowed packages
+     * @deprecated use getAllowedPackages()
+     */
+    @Deprecated(forRemoval = true)
```

Comment:
My bad, `forRemoval` is not available in Java 8
```suggestion
    @Deprecated
```